modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -2,7 +2,7 @@
 	if(stat != CONSCIOUS) // you're not The One
 		return ..()
 	// Cat-people always land on their feet
-	if(isfelinid(src) || istajaran(src))
+	if(isfeline(src))
```

Comment:
The reason why  the Feline Traits quirk was free was especially because it didn't really feature any of the upsides of being a Felinid.

I feel like making it so they get the upsides too kind of entirely defeats the point of having Felinids as a separate race.